Various updates

drazik opened the pull request:
Some little things are grouped in this PR:

* Fix a typo in the nav in FR: « Securité » => « Sécurité »
* Fix the padding of the content: since we have an `overflow-y: scoll` on the content element, the padding-bottom was not rendered. Adding a intermediate element with the padding fixes it
* Update app name to "Pass" for both FR and EN
* Use a logo with the "beta" word
* Run tests in travis builds

drazik committed:
fix(locales): Typo in nav

drazik committed:
fix(App): Padding in content

On an element with `overflow-y: scroll`, the padding-bottom is ignored.
To have a padding in the `<Content />`, we should add another element
inside it and apply the padding on it.

drazik committed:
feat: Update app name to "Pass"

drazik committed:
feat: Use beta logo

drazik committed:
chore: Run test in travis builds

ptbrowne commented on src/components/App.jsx:
```diff
@@ -48,35 +48,37 @@ export const DumbApp = ({ breakpoints: { isDesktop } }) => {
-                  <Redirect from="*" to="/presentation" />
-                </Switch>
+              <Content>
+                <div className="u-p-2 u-mt-auto u-mb-auto">
```
Are we sure we do not want a margin-bottom here ? I have a bit of trouble understanding why the password app should have a custom layout.

drazik replied:
The password app has its content vertically centered. That's why we have auto vertical margins.

ptbrowne replied:
Are you sure that the password app will have all its pages with vertically centered content ? I think this is a coincidence that all the pages, as of now, are vertically centered. IMHO, the padding and vertical centering should not be hard coded in the general layout of the app, but preferably in a wrapper for all views.

drazik replied:
I see your point, you're right. I'm doing this change right now.

drazik replied:
See https://github.com/cozy/cozy-passwords/pull/15/commits/597ac9498d8cc098e4e7150618ef28fdcd8a074e

ptbrowne replied:
Thanks !

drazik committed:
refactor: Wrap pages in VerticallyCentered component

Today all pages have their content vertically centered, but it may not
be the case for future ones.
